OpenRA.Mods.Common/ActorExts.cs

```diff
@@ -80,6 +80,39 @@ public static Target ResolveFrozenActorOrder(this Actor self, Order order, Color
 			return Target.Invalid;
 		}
 
+		public static void PlayVoice(this Actor self, Actor actor, string phrase, string variant)
```

Comment:
The `actor` parameters here and below aren't used (and don't make a lot of sense?).  Could you please remove them?